### PR TITLE
New memberships methods

### PR DESCRIPTION
**Pull request by chregu**

### CODE REVIEW

File: cmd/rokka/cli/membership.go [hunk 1]
```diff
@@ -2,24 +2,60 @@ package cli
...
+	if err != nil {
+		return nil, err
+	}
+	return c.CreateNewMembershipWithCurrentUser(args[0], roles)
+}
+
+func listMembership(c *rokka.Client, args []string) (interface{}, error) {
+	if len(args) == 2 {
+		return c.ListMembershipForUUID(args[0], args[1])
+
```
Comment (mweibel) [inline cmd/rokka/cli/membership.go]:
```suggestion
```

File: cmd/rokka/cli/users.go [hunk 2]
```diff
@@ -6,7 +6,15 @@ import (
 )
 
 func createUser(c *rokka.Client, args []string) (interface{}, error) {
-	return c.CreateUser(args[0], args[1])
+	if len(args) == 2 {
+		return c.CreateUser(args[1], args[0])
+
```
Comment (mweibel) [inline cmd/rokka/cli/users.go]:
```suggestion
```

**Review by mweibel — COMMENTED** [review]
A few minor things but otherwise looks good, IMO :) 

Thanks!

File: cmd/rokka/cli/membership.go (hunk 1 above)
Comment (chregu) [reply cmd/rokka/cli/membership.go]: fixed

File: cmd/rokka/cli/users.go (hunk 2 above)
Comment (chregu) [reply cmd/rokka/cli/users.go]: fixed

**Review by mweibel — APPROVED** [review]

**Pull request merged**